docs/developer/CONTRIBUTING_HERE.md

```diff
@@ -0,0 +1,31 @@
+# Contributing to jupyter/accessibility directly
+
+This contribution guide is for people who want to add knowledge to the jupyter/accessibility repository. If you are instead looking to contribute to accessibility in the Jupyter ecosystem, see [here](../../CONTRIBUTING.md).
+
+Before making any contributions, read through [ARCHITECTURE.md](../../ARCHITECTURE.md) to understand the different directories and files present in this repo.
+
+## Core Workflow
+
+1. get a free GitHub account
+2. if you are contributing to an existing ticket/issue
+    * it's best to let the community know so we can support each other
+	* comment your interest on the original issue (best practice)  
+3. if your issue/ticket does not exist, open a new one [here](https://github.com/jupyter/accessibility/issues)
+    * describe your issue or proposal with as much detail as possible
+4. fork this repository
+5. create a well named branch
+6. develop the solution to the issue/ticket
+7. make one (or more) pull requests to this repository
+8. gather and discuss findings
+9. repeat steps 2+ as needed
+10. please don't hesitate to reach out for help and guidance
+
+
+## Communication
+
+At this time, you are free to communicate through open GitHub issues. Some individual team members may also be available via Gitter.
+
+## Useful links
```

Comment:
Is this section intended to be empty? 
if there is no content it would be best to remove to avoid confusion